src/edu/csus/ecs/pc2/ui/board/ScoreboardCommon.java

```diff
@@ -102,6 +189,11 @@ public void generateOutput(String xmlString, String xslDir, String outputDir, Lo
                 } catch (Exception e) {
                     log.log(Log.WARNING, "Trouble transforming " + xslFilename, e);
                 }
+            } else {
+                if (new File(xslDir + File.separator+ xslFilename).isFile()) {
+                    // not xsl, and a file, so just copy it**
```

Comment:
What does the trailing ** mean?  "copy it**" ?

Reply:
** means only if needed :)